doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -33,7 +33,7 @@ Build instructions are available for the following platforms:
 - [Linux :penguin:](#linux)
 - [AIX :blue_book:](#aix)
 - [Windows :ledger:](#windows)
-- [macOS :apple:](#macos)
+- [MacOS :apple:](#MacOS)
```

Comment:
Looking at https://www.apple.com/ca/macos/catalina/, the original spelling is correct.

Reply:
Ok, I'll switch all references to 'macOS'; I started by correcting the link from 'macos' which targeted 'macOS', but I should have checked on the official spelling.

Reply:
Done.